library/src/schemas/any/any.ts

```diff
@@ -17,24 +21,10 @@ export type AnySchema<TOutput = any> = BaseSchema<any, TOutput> & {
  */
 export function any(pipe: Pipe<any> = []): AnySchema {
   return {
-    /**
-     * The schema type.
-     */
-    schema: 'any',
-
-    /**
-     * Whether it's async.
-     */
+    kind: 'any',
     async: false,
+    pipe,
 
-    /**
```

Comment:
Is there a reason why you removed this comment everywhere?

Reply:
All of the comments for the return types were moved to the base interface in the `types.ts` file. IntelliSense surfaces it everywhere in the codebase, so it's redundant to have them in each schema definition.

For properties specific to individual schemas, the comments relevant to those properties were moved to the type definition at the top of the file.

Reply:
Ok, you are right, for documentation and tooling with IntelliSense it is actually better this way. I had added the comments to the return object of the schemas at that time to make the code more understandable when looking at a single schema file on GitHub. How would you proceed here?

Reply:
I don't think it's necessary to add the extra comments everywhere. The codebase itself is relatively straight-forward to navigate. Once you have documentation added to the website it will make it easier to understand that patterns each of these functions follow as well.

Reply:
Thank you for your feedback. I will think about it and then make a decision.

Reply:
I have thought about this and would like to keep the comments for now, as I think they make the code more understandable.

Reply:
I would advise against it. You're adding unnecessary maintenance burden to yourself to repeat these comments throughout the codebase. IntelliSense will surface the doc comments from the root `BaseSchema` type when a contributor is editing any of the schemas.

Reply:
The reason I want to keep the comments is mainly so that people looking at a single schema on GitHub can immediately understand where to find what. The effort on my part is not high, as new APIs are added only occasionally. I would rather have one comment too many than one too few.